Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/DirectoryInitialization.java

```diff
@@ -70,10 +70,17 @@ private static void init(Context context)
         if (setDolphinUserDirectory(context))
         {
           initializeInternalStorage(context);
-          initializeExternalStorage(context);
+          boolean wiimoteIniWritten = initializeExternalStorage(context);
           NativeLibrary.Initialize();
           NativeLibrary.ReportStartToAnalytics();
 
+          if (wiimoteIniWritten)
+          {
+            // This has to be done after calling NativeLibrary.Initialize(),
+            // as it relies on the config system
+            EmulationActivity.updateWiimoteNewIniPreferences(context);
```

Comment:
Does this need to be passed to `AfterDirectoryInitializationRunner.run`? I believe that is how we make sure this starts after `NativeLibrary.Initialize` completes.

Reply:
That would make sense if we were outside of `DirectoryInitialization.init`, but since we're inside it, we just have to make sure that this is called after `NativeLibrary.Initialize`. (The call to it is just a few lines above this line.)